docs/INSTALL.SSL.md

```diff
@@ -162,9 +162,26 @@ file.  For e.g. on Ubuntu, if you installed ovn-controller via the package
 ```
 OVN_CTL_OPTS="--ovn-controller-ssl-key=/etc/openvswitch/ovncontroller-privkey.pem  --ovn-controller-ssl-cert=/etc/openvswitch/ovncontroller-cert.pem --ovn-controller-ssl-ca-cert=/etc/openvswitch/cacert.pem"
 ```
+Now, when you start the ovnkube utility on  with "--init-cluster-manager",
+you should pass the SSL certificates to it. For e.g:
 
-Now, when you start the ovnkube utility on master, you should pass the SSL
-certificates to it. For e.g:
+```
+sudo ovnkube -k8s-kubeconfig kubeconfig.yaml -loglevel=4 \
+ -k8s-apiserver="http://$CENTRAL_IP:8080" \
+ -logfile="/var/log/ovn-kubernetes/ovnkube.log" \
+ -init-cluster-manager="$NODE_NAME" -cluster-subnets=$CLUSTER_IP_SUBNET \
+ -k8s-service-cidr=$SERVICE_IP_SUBNET \
+ -nb-client-privkey /etc/openvswitch/ovncontroller-privkey.pem \
+ -nb-client-cert /etc/openvswitch/ovncontroller-cert.pem \
+ -nb-client-cacert /etc/openvswitch/cacert.pem \
+ -nb-cert-common-name ovncontroller \
+ -sb-client-privkey /etc/openvswitch/ovncontroller-privkey.pem \
+ -sb-client-cert /etc/openvswitch/ovncontroller-cert.pem \
+ -sb-client-cacert /etc/openvswitch/cacert.pem \
+ -sb-cert-common-name ovncontroller 
```

Comment:
Do we need all the above input arguments related to nb and sb to run cluster manager?